WordPressKitTests/WordPressOrgRestApiTests.swift

```diff
@@ -63,4 +63,34 @@ class WordPressOrgRestApiTests: XCTestCase {
         }
         waitForExpectations(timeout: 2, handler: nil)
     }
+
+    func testSuccessfulPostCall() {
+        stub(condition: isAPIRequest()) { _ in
+            let stubPath = OHPathForFile("wp-reusable-blocks.json", type(of: self))
+            return fixture(filePath: stubPath!, headers: ["Content-Type" as NSObject: "application/json" as AnyObject])
+        }
+        let expect = self.expectation(description: "One callback should be invoked")
+        let api = WordPressOrgRestApi(apiBase: apiBase)
+        let blockContent = "<!-- wp:paragraph -->\n<p>Some text</p>\n<!-- /wp:paragraph -->\n\n<!-- wp:list -->\n<ul><li>Item 1</li><li>Item 2</li><li>Item 3</li></ul>\n<!-- /wp:list -->"
```

Comment:
This is a long value to use in the test given that the response is stubbed. Is there a particular reason for use it? Having an example with real-world-like input?

Reply:
We could make it shorter, the only condition is to match the content from the stub:
https://github.com/wordpress-mobile/WordPressKit-iOS/blob/e3fe762af77721c637c0c8e00f966bffdc16e134/WordPressKitTests/Mock%20Data/wp-reusable-blocks.json#L20